cafebabel/static/css/styles.css

```diff
@@ -447,6 +442,10 @@ button:hover,
     transform: translateY(2.5rem);
 }
 
+.authentication-form > div > input:-webkit-autofill {
```

Comment:
Please comment where you got this from with the link ;)

Reply:
ok